packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -3765,27 +3767,19 @@ export default function UpdateCompositeDogForm(props) {
             }
           });
           compositeToysToUnLink.forEach((original) => {
-            try {
-              promises.push(
-                DataStore.save(
-                  CompositeToy.copyOf(original, (updated) => {
-                    updated.compositeDogCompositeToysName = null;
-                    updated.compositeDogCompositeToysDescription = null;
-                  })
-                )
+            if (!canUnlinkCompositeToys && compositeToysToUnLink.length > 0) {
+              throw Error(
+                \`CompositeToy \${original.id} cannot be unlinked from CompositeDog because compositeDogCompositeToysName is a required field.\`
```

Comment:
Also, not every model has "id". You can look at the `primaryKeys` array attached to each model and just take the first one -- should be compositeToysToUnlink[0].kind or something.

Reply:
Got it. Will address